pkgs/tools/system/kanata/default.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub
+, lib
+, libevdev
+, pkg-config
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kanata";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "jtroo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-sL9hP+222i8y0sK3ZEx66yXBTgZp5ewoPUlZS4XnphY=";
+  };
+
+  cargoHash = "sha256-uhN1UdwtU0C0/lpxUYoCcMLABFTPNO5wKsIGOBnFpzw=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libevdev ];
+
+  meta = with lib; {
+    description = "A cross-platform advanced keyboard customization tool";
+    homepage = "https://github.com/jtroo/kanata";
+    license = licenses.lgpl3;
```

Comment:
You need to also change that according to [this](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206).